src/Rips_complex/include/gudhi/Sparse_rips_complex.h

```diff
@@ -108,17 +164,17 @@ class Sparse_rips_complex {
       complex.expansion(dim_max);
       return;
     }
-    const int n = boost::size(params);
-    std::vector<Filtration_value> lambda(n);
+    const Vertex_handle n = num_vertices(graph_);
```

Comment:
`Vertex_handle` is an int, whereas `num_vertices` returns a `std::size_t`

Reply:
Well, yes, we use int for vertices, and vector uses size_t for its size, so there has to be a conversion somewhere...

Reply:
Hmm, the way I use i and n, they could actually just be size_t, I'll do that.

Reply:
Done. Note that it is broken that we arbitrarily define Vertex_handle as int in Sparse_rips, but the 2-function design makes that a bit complicated to avoid.